crates/goose/src/lib.rs

```diff
@@ -1,6 +1,8 @@
 pub mod agents;
+pub mod config;
 pub mod key_manager;
```

Comment:
i think this can be removed? key_manager is still used in `routes/secrets.rs` but can be replaced with config